documentation/using-spark/StyledComponents.mdx

```diff
@@ -266,6 +267,43 @@ function RoundedCard({ kind, className }: RoundedCardProps) {
 }
 ```
 
+## "as" polymorphic prop
+
+Coming from a **CSS-in-JS** solution, you're probably used to the `as` prop.
+
+This prop allows you to keep all the styling you've applied to a component but just switch out what's being ultimately rendered (be it a different HTML tag or a different custom component).
+
+- For example, with **styled-components**, you could do something like this:
+
+```tsx
+import { Button } from 'ui/buttons'
+import { Link } from 'ui/links'
+
+function Component() {
+  return (
+    <Button as={Link} route="someRoute">
+      some link that looks like a Button
+    </Button>
+  )
+}
+```
+
+On our end, since we are using Radix, to address these use cases, we use their `asChild` prop. Check out their [guide here](https://www.radix-ui.com/docs/primitives/guides/composition).
```

Comment:
Perhaps something like this:
On our end, since we decided to follow Radix approach, to address these use cases, we use the `asChild` prop. Check out their [guide here](https://www.radix-ui.com/docs/primitives/guides/composition).